package.json

```diff
@@ -45,6 +45,9 @@
     "test": "npm run build && npm run test:unit && npm run test:types",
     "test:unit": "node ./test/all.test.js",
     "test:types": "tsd",
+    "test:smoke:bun": "bun test ./test/smoke/bun.test.js",
+    "test:smoke:node14": "node --experimental-modules ./test/smoke/node-esm.test.js",
```

Comment:
Lets drop node14?

Reply:
We have to do it due to deps requirements :-(